erigon-lib/state/inverted_index_test.go

```diff
@@ -104,10 +104,10 @@ func TestInvIndexCollationBuild(t *testing.T) {
 
 	bs, err := ii.collate(ctx, 0, roTx)
 	require.NoError(t, err)
-	require.Equal(t, 3, len(bs))
-	require.Equal(t, []uint64{3}, bs["key2"].ToArray())
-	require.Equal(t, []uint64{2, 6}, bs["key1"].ToArray())
-	require.Equal(t, []uint64{6}, bs["key3"].ToArray())
+	//require.Equal(t, 3, len(bs))
```

Comment:
let's restore this unit test somehow?

Reply:
There is correct check below:
```go
	require.Equal(t, []string{"key1", "key2", "key3"}, words)
	require.Equal(t, [][]uint64{{2, 6}, {3}, {6}}, intArrs)
```

Correct way to check collation is to compress file, read written keys and decode each ef, then compare ordering and contents which is made later - after `ii.buildFiles` where collation is compressed.